jme3-examples/build.gradle

```diff
@@ -19,8 +19,8 @@ dependencies {
     implementation project(':jme3-effects')
     implementation project(':jme3-jbullet')
     implementation project(':jme3-jogg')
-//    implementation project(':jme3-lwjgl')
-    implementation project(':jme3-lwjgl3')
+    implementation project(':jme3-lwjgl')
```

Comment:
@JNightRide ,
Can you explain why using jme3-lwjgl3 instead of jme3-lwjgl solves the problem?

Reply:
I didn't understand the question ("using jme3-lwjgl3 instead of jme3-lwjgl"), this part changes from **jme3-lwjgl3 to jme3-lwjgl** to solve the problem.

Currently jme3 uses jme3-lwjgl3 in the examples module.

jme3-lwjgl3 uses runtimeOnly "..." to implement its dependency (which is fine to run with maven or directly), when building the project it fails to export the jar files and is left without binaries.

jme3-lwjgl by default includes jar files (binary):
```
    /*
     * Upgrades the default jinput-2.0.5 to jinput-2.0.9 to fix a bug with gamepads on Linux.
     * See https://hub.jmonkeyengine.org/t/linux-gamepad-input-on-jme3-lwjgl-splits-input-between-two-logical-gamepads
     */
    api 'net.java.jinput:jinput:2.0.9'
    api 'net.java.jinput:jinput:2.0.9:natives-al
```